examples/using-type-definitions/gatsby-node.js

```diff
@@ -0,0 +1,82 @@
+exports.addTypeDefs = ({ addTypeDefs }) => {
```

Comment:
BTW, we use `create` rather than `add` as our operator for making a new thing https://www.gatsbyjs.org/docs/api-specification/#operators